Assets/Tests/koob_Assets/Scripts/Skill.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// スキル用クラス
+/// </summary>
+public class Skill
+{
+    int damage;
+    string name;
+
+    public Skill(int _damage ,string _name)
+    {
+        damage = _damage;
```

Comment:
おそらく同じ変数名になるため実引数名の先頭に`_`をつけているのだと思いますが、下記のようにすれば同じ変数名でも大丈夫です。
```suggestion
        this.damage = damage;
```